crypto/cryptonight_aesni.h

```diff
@@ -327,7 +327,6 @@ void cryptonight_hash(const void* input, size_t len, void* output, cryptonight_c
 	__m128i division_result_xmm = _mm_cvtsi64_si128(h0[12]);
 	__m128i sqrt_result_xmm = _mm_cvtsi64_si128(h0[13]);
 
-	_control87(RC_DOWN, MCW_RC);
```

Comment:
You can't just remove _control87, it sets FPU rounding mode to "round down" aka "round to negative infinity". It must be replaced with `std::fesetround(FE_DOWNWARD);`